src/pages/index.svelte

```diff
@@ -21,8 +21,9 @@
   import Step3 from '../images/step-3.svg';
 
   function handleLearnMoreClick() {
-    const stepsSection = document.getElementById('steps-section');
-    stepsSection.scrollIntoView({ behavior: 'smooth' });
+    var navBarHeight = 70; // --height-nav: 7rem;
+    var topOfStepsSection = document.getElementById('steps-section').offsetTop - navBarHeight;
```

Comment:
Please use `const` here instead of var, and get the height from the css variable like this, for example (untested):
```
getComputedStyle(document.documentElement).getPropertyValue('--heightNav')
```
This way we can update the nav styles without breaking this code.